examples/spiffs/spiffs_example.c

```diff
@@ -77,6 +85,8 @@ static void example_fs_info()
 void test_task(void *pvParameters)
 {
     esp_spiffs_init();
+    // esp_spiffs_init(0x200000, 0x1000);   // for run-time configuration
```

Comment:
Could this not be surrounded by ```#if SPIFFS_SINGLETON == 1``` or perhaps the comment could refer to the singleton configuration.